incubator/exporter-prometheus/src/main/java/io/aklivity/zilla/runtime/exporter/prometheus/internal/processor/MetricsProcessor.java

```diff
@@ -41,7 +41,7 @@
     private static final long[][] EMPTY = new long[0][0];
 
     private final Map<Metric.Kind, List<MetricsLayout>> layouts;
-    private final LabelManager labels;
+    private final IntFunction<String> localNameSupplier;
     private final Function<String, String> kindSupplier;
     private final Function<String, String> nameSupplier;
     private final Function<String, String> descriptionSupplier;
```

Comment:
In general, we tend to name these functional interface variables following a pattern like `supplyKind` rather than `kindSupplier`.

Please rename these fields and the constructor parameters accordingly.

Reply:
done